docs/meshstack.identity-import.md

```diff
@@ -0,0 +1,6 @@
+---
+id: meshstack.ldap-identity-import
+title: LDAP Identity Import
+---
+
+meshStack supports importing users and groups into meshStack from a source that supports the LDAP protocol, for example, Microsoft Active Directory. The `Simple Paged Results Control` is used to do a paginated LDAP query to fetch the users and groups. Because of this, it is required that the LDAP server supports the `Simple Paged Results Control`. Other than importing users and groups, meshStack also has the capability to assign a [customer role](./meshcloud.customer.md#assign-meshcustomer-roles) to the imported group. The entities read via LDAP is imported to meshStack via the [meshObject import API](./meshstack.api.md#meshobject-api).
```

Comment:
d: I think you should also mention the regex we use to determine tags and also the option to identify partner admin role? So in general you should add the dhall snippet documentation here for the Identity Import service.

Reply:
Yes. But the dhall configuration wasn't complete the last time I looked at it. @felixzieger was going to look into it I think. But probably we will need a few changes to the dhall config to add examples. Not sure if we can consider that to be within the ticket.

Reply:
I am adding the missing bits in [another PR](https://github.com/meshcloud/meshfed-release/pull/309). I will add dhall configuration snippets (haven't done that before, seems like a good learning opertunity). I think the easiest way would be wait until @pubudug has finished work on the docs and then start on a new branch (referencing the same ticket).

Reply:
I will resolve this because the requirements are also volatile right now and the config can change based on the information we get from CoBa.